README.md

```diff
@@ -229,6 +229,30 @@ The name should not contain whitespaces or any other special characters due to f
       end
     end
 
+## Additional documentation
+
+## Setting a Swagger defaultValue
+
+Grape allows for an additional documentation hash to be passed to a parameter.
+
+    params do
+      requires :id, type: Integer, desc: 'Coffee ID'
+      requires :temperature, type: Integer, desc: 'Temperature of the coffee in celcius', documentation: { example: 72 }
+    end
+
+The example parameter will populate the Swagger UI with the example value, and can be used for optional or required parameters.
+
+Grape uses the option `default` to set a default value for optional parameters. This is different in that Grape will set your parameter
+ to the provided default if the parameter is omitted, whereas the example value above will only set the value in the UI itself. This will
```

Comment:
Super minor. Just make this one line while you're at it, don't break at 80 characters.